tests/Fixer/LanguageConstruct/IsNullFixerTest.php

```diff
@@ -224,7 +224,7 @@ public function provideFixCases()
                 '<?php if ((is_null($u) or $v) and ($w || is_null($x)) xor (!is_null($y) and $z)) echo "foo"; ?>',
             ],
 
-            // edge cases: contains dangerous constructs, wrapped into parentheses
+            // edge cases: $isContainingDangerousConstructs, $wrapIntoParentheses
```

Comment:
@keradus I guess you want to revert this back, and instead make the opposite change on line 315

Reply:
I spot from where you took those vars, I didn't saw it in prev PR